frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -163,14 +166,42 @@ export class ClusterOverview extends React.Component {
     }));
   }
 
-  getPrometheusMetrics() {
+  async _getStorageMetrics() {
+    try {
+      let queryTotal = CAPACITY_STORAGE_TOTAL_DEFAULT_QUERY;
+      let queryUsed = UTILIZATION_STORAGE_USED_DEFAULT_QUERY;
+
+      const metrics = await this.getPrometheusMetrics();
+      if (_.get(metrics, 'data', []).find(metric => metric === 'kubelet_volume_stats_capacity_bytes')) {
```

Comment:
is this check correct ? Its a metric that was used before. We should rather check for presence of one of new ceph metrics right ?

Reply:
oh, good catch. I'll parametrize that to avoid similar situation in the future